src/plone/restapi/interfaces.py

```diff
@@ -1,4 +1,9 @@
 # -*- coding: utf-8 -*-
+
```

Comment:
The pylint-ignores are only necessary in order to comply with how zope.interface works:
zope.interface requires method declarations to not have the `self` attribute, but pylint isn't happy about that:
> E0211: Method has no argument
> W0221: Arguments number differs from overridden '__call__' method

I think we should not ignore those globally, because in "normal" code those cheks make perfectly sense, but it just does not work with zope.interface; that's why I've ignored them here locally.